### ISSUE TITLE
Refreshing page causes 'graptcha not defined' error

### ISSUE THREAD

**Issue by bmills22** [appleboy/react-recaptcha#124]
I am using react-router in my project with redux, and everything works when navigating normally. However when a user refreshes the page with the captcha on it, 3/4 times I get:
`grecaptcha is not defined`

I think this is race condition because, when I work locally, I do not get this error. Here's how I have my code setup:
**index.html**
```
<head>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" async defer></script>
</head>
```

**Signup.jsx**
```...
<Recaptcha {...captcha} sitekey='xxx-xxx-xxx'  
     render = 'explicit'  
     verifyCallback={this.verifyCallback}  
     data-size="compact"  
     onloadCallback={this.captchaLoadCallback}  
     style={this.captchaStyle}  
     size='small' />  
...
```

Any ideas?

**Comment by lostpebble** [comment]
I experienced the same problem. I'm working with server rendering, which I thought was the issue (maybe the component was being created too early on the server and couldn't initialise properly client side). So I tried not rendering the component from the server and only on the client -BUT that still didn't fix the error.

As you say, there are probably some race conditions somewhere. A dirty fix which is working for me now was pretty much this (inside whatever parent component renders the Recaptcha):

```
componentDidMount() {
	window.setTimeout(() => this.setState({ renderRecaptcha: true }), 1000);
}
```

Basically, waiting one second before rendering the component:

```
<div className="recaptcha">
	{this.state.renderRecaptcha ?
		<Recaptcha
			sitekey="xxxx"
			render="explicit"
			verifyCallback={this.verifyRecaptcha}
			onloadCallback={this.loadedRecapcha}
		/> : null}
</div>
```
